### PR TITLE
Added colored loading bar to app

### LINKED ISSUE #84: Add loading animation when navigating the app
See how IG shows a loading animation on the top of the viewport to indicate loading.

There is an easy way to do this, we can discuss it in a call.

### PR DESCRIPTION
**Pull request by alshakerM**
Fixes #84

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -29,3 +29,33 @@ textarea {
 .MuiDialogContent-root {
   padding: 0 20px;
 }
+
+.loading-bar {
+  height: 5px;
```
Comment (alshakero) [inline styles/globals.css:34]: Seems like 3px is enough.

```suggestion
  height: 3px;
```

File: pages/_app.js [hunk 2]
```diff
@@ -9,13 +10,32 @@ const routeTitleMap = {
...
+    };
+    router.events.on('routeChangeStart', handleStart);
+    router.events.on('routerChangeComplete', handleEnd);
+
+    return () => {
+      router.events.off('routeChangeStart', handleStart);
+      router.events.off('routerChangeComplete', handleEnd);
+    };
+  }, [router]);
+  console.log(isLoading);
```
Comment (alshakero) [inline pages/_app.js:30]:
```suggestion

```

**Review by alshakero — APPROVED** [review]
Beautiful. Looks great.

File: styles/globals.css (hunk 1 above)
Comment (alshakerM) [reply styles/globals.css:34]: Copy that

**Pull request merged**